securedrop/journalist_templates/base.html

```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ g.organization_name }}</title>
+  <title>{{ get_descriptive_title() }} | {{ g.organization_name }}</title>
```

Comment:
I _suppose_ that the `Context` argument gets passed automatically by Jinja? I don't actually know about that @saffronsnail, maybe you do!

Reply:
Yeah, I wrote the type hint that way because the `pass_context` generator adds it automatically.

Reply:
Gotcha. Does **mypy** complain if you add the type hint but don't pass the argument explicitly? If that's the case, I think it's fair to `# type: ignore [not-sure-what-the-error-name-is]` where most relevant.

Reply:
**Note** the space between `ignore` and `[...]` :wink: 

For line length: `# noqa: E501`  it needs to go _after_ the type ignore comment, e.g. `# type: ignore [has-type]  # noqa: E501`.

Reply:
Oh, I moved the comment to just above the function before seeing this comment. If you'd prefer that the note is in the same line as the ignore directive, I can change it to use noqa.

Reply:
Oh, it looks like I'll need to do that anyway because just having the type directive makes it too long.

Reply:
I'm still leaving the comment above the function because I think it's nice for human-intended context to not be hidden past the edge of the screen, but still let me know if that should be moved for some reason.

Reply:
Any syntax that work seems good to me for this, and you've got a good reason to pick the one you chose. :+1:

Reply:
Ok, so it looks like I get a syntax error if I specify both the mypy return code and the noqa return code, but if I only specify one of them it's fine for some reason. `securedrop/journalsit_app/sessions.py` only specifies the type code, but `securedrop/store.py` only specifies the noqa code, so it's not clear which one is preferred. I'm about to upload a commit that specifies type but I can switch it around if needed.

Reply:
I just made a suggestion that should unlock you @saffronsnail. Sorry we're spread across multiple threads :face_with_spiral_eyes:

Reply:
Aah, you know what, I didn't think to refresh the page as I was typing in updates so I didn't see some of your replies. Still getting used to GitHub. :sweat_smile:

It turns out I was putting the directive in the wrong spot: the error occurs on the line that returns the value, not on the function signature. Putting it on the correct line deals with the long line problem without any directives. Unfortunately, `flake8` reports a syntax error when the error code is specified in a `type: ignore` directive, so I told it to ignore F723. 

`make test` passes locally now (*with* the space!), hopefully the server agrees.